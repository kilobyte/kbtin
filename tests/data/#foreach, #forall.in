#foreach {a b c} {#showme}
#foreach {A {1 2} C} {#showme}
#foreach {a b c} {#showme x$0x}
#forall 1 {#showme}
