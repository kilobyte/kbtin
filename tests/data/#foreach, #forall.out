a
b
c
A
1 2
C
xax
xbx
xcx
1
